spec/models/video_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
+require 'shoulda/matchers'
```

Comment:
This was not in the solution but is in the shoulda-matchers readme file on github